examples/apps/color_finder.py

```diff
@@ -0,0 +1,740 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017 Anki, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the file LICENSE.txt or at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS, 
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Cozmo looks around and drives after colors.
+
+When the program starts, Cozmo will look around for the color blue.
+Tap the cube illuminated blue to switch Cozmo's target color between blue, red, green, and yellow.
+Tap the blinking white cube to have the viewer display Cozmo's pixelated camera view.
+
```

Comment:
No blank line needed before the end of the docstring. This applies to all docstrings in the file.